tests/unit/test_utils.py

```diff
@@ -114,41 +96,11 @@ def test_check_if_empty_file_parquet(caplog):
         pass
 
     with caplog.at_level(logging.WARNING):
-        check_if_empty_file(
-            path=EMPTY_PARQUET_PATH, if_empty="warn", file_extension=".parquet"
-        )
+        check_if_empty_file(path=EMPTY_PARQUET_PATH, if_empty="warn")
         assert f"Input file - '{EMPTY_PARQUET_PATH}' is empty." in caplog.text
     with pytest.raises(ValueError):
-        check_if_empty_file(
-            path=EMPTY_PARQUET_PATH, if_empty="fail", file_extension=".parquet"
-        )
-    with pytest.raises(SKIP):
-        check_if_empty_file(
-            path=EMPTY_PARQUET_PATH, if_empty="skip", file_extension=".parquet"
-        )
-
-    os.remove(EMPTY_PARQUET_PATH)
-
-
-def test_check_if_empty_file_one_column_parquet(caplog):
```

Comment:
Can you pls add a test for the case with a parquet file with columns but no data?